### PR TITLE
Add more examples

### PR DESCRIPTION
**Pull request by arjxn-py**
Towards #502

### CODE REVIEW

File: examples/ArchDetail.jcad
Comment (martinRenou) [inline examples/ArchDetail.jcad:1]: There may be many objects we can remove from that file to simplify it without changing the visual result

File: examples/Gear.jcad [hunk 1]
```diff
@@ -978,7 +950,7 @@
...
...0.451839026756215\n2  954 3 29 0 0.451839026756215\n2  955 305 33 0 0.451839026756215\n2  956 305 35 0 0.451839026756215\n2  957 3 36 0 0.451839026756215\n2  958 306 40 0 0.451839026756215\n2  959 306 42 0 0.451839026756215\n2  960 3 43 0 0.451839026756215\n0\n\n0101000\n+120 0 -118 0 *\nVe\n1e-07\n4.502456 -10.389524 0\n0 0\n\n0101101\n*\nEd\n 1e-07 1 1 0\n1  61 0 0 0.466647834024772\n2  961 307 5 0 0.466647834024772\n2  962 307 7 0 0.466647834024772\n2  963 282 8 0 0.466647834024772\n2  964 3 15 0 0.466647834024772\n2  965 308 19 0 0.466647834024772\n2  966 308 21 0 0.466647834024772\n2  967 3 22 0 0.466647834024772\n2  968 309 26 0 0.466647834024772\n2  969 309 28 0 0.466647834024772\n2  970 3 29 0 0.466647834024772\n2  971 310 33 0 0.466647834024772\n2  972 310 35 0 0.466647834024772\n2  973 3 36 0 0.466647834024772\n2  974 311 40 0 0.466647834024772\n2  975 311 42 0 0.466647834024772\n2  976 3 43 0 0.466647834024772\n0\n\n0101000\n+118 0 -116 0 *\nVe\n1e-07\n3.985266 -8.913094 0\n0 0\n\n0101101\n*\nEd\n 1e-07 1 1 0\n1  62 0 0 1.56439478425364\n2  977 312 5 0 1.56439478425364\n2  978 312 7 0 1.56439478425364\n2  979 282 8 0 1.56439478425364\n2  980 3 15 0 1.56439478425364\n2  9
```
Comment (martinRenou) [inline examples/Gear.jcad:953]: Shouldn't you remove this property entirely?

Comment (martinRenou) [reply examples/Gear.jcad:953]: I mean not leaving `"Shape": ""` in the file

Comment (arjxn-py) [reply examples/Gear.jcad:953]: When i did that the last time, it made objects invisible.

**Pull request merged**